src/controllers/state_controller.hpp

```diff
@@ -661,13 +658,21 @@ void AerState::initialize() {
 
   state_->initialize_qreg(num_of_qubits_);
   state_->initialize_creg(num_of_qubits_, num_of_qubits_);
-  rng_.set_seed(seed_);
```

Comment:
Do we still need to track the `seed_` member after this change?  I see it's still used in `clear_ops()`, but it's not 100% clear to me why that function should reset the RNG.  Admittedly, I'm not exactly clear on the circumstances in which it's called.

Reply:
https://github.com/Qiskit/qiskit-aer/blob/b5b0c4dc271e8a986b7cd7d1fe3d40e3862b1d51/src/controllers/state_controller.hpp#L1228

This `seed_` is to be returned as a member of a result. `AerState.last_result()` returns it. We may not need to construct a result as `AerSimulator` does for each call of methods, but it is not expensive to keep `seed_`, I think.